sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -1987,6 +1987,13 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
     }
   }
 
+  test("SPARK-25918: LOAD DATA LOCAL INPATH should handle a relative path") {
```

Comment:
This does not belong to this test suite. `HiveCommandSuite.scala` is the best place, although this is not for hive module.

Reply:
Ya. I know. I put this here before the previous test case are [here](https://github.com/apache/spark/pull/22927/files/efb99da8fb505aaeeb0d95fff99c245bd3c0a0b8#diff-1ea02a6fab84e938582f7f87cc4d9ea1R1997).

Reply:
I'll move this.